src/components/Tracking.tsx

```diff
@@ -37,32 +37,32 @@ export interface TrackingProps {
 
 const HORIZONTAL = ['bottom', 'top'];
 
-// function infoFromWheel(
-//   event: React.WheelEvent<SVGRectElement>,
-//   axisContext: Record<string, PlotAxisContext>,
-//   plotHeight: number,
-// ) {
-//   const ratio = 1 + event.deltaY * -0.001;
+function infoFromWheel(
+  event: WheelEvent,
+  axisContext: Record<string, PlotAxisContext>,
+  plotHeight: number,
+) {
```

Comment:
I think I'd prefer if we could implement the "mouse wheel to zoom" action using the data returned by `infoFromMouse`. We can add more info to it if necessary, but the API will be simpler if all mouse events generate the same info.

Reply:
We don't have yDelta in mouseEvent 
But yes i will try to fix make it simpler

Reply:
I see. Then can you try to make a common interface that has the event type as a parameter?
Something like `TrackingResult<EventType extends MouseEvent = MouseEvent>`